esptfa_arima/Authentication/authentication.py

```
@@ -0,0 +1,42 @@
+            )
+            if raw_token:
+                logger.debug("JWT token found in cookie.")
+
+        if raw_token is None:
+            return None
+
+        try:
+            validated_token = self.get_validated_token(raw_token)
+            user = self.get_user(validated_token)
+            logger.debug(f"Successfully authenticated user {user.username} via JWT.")
+            return user, validated_token
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)